File: python/main2.py
from BinarySearchTree import BinarySearchTree
from Node import Node

tree = BinarySearchTree()

tree.insert(Node(12))
tree.insert(Node(5))
tree.insert(Node(2))
tree.insert(Node(3))
tree.insert(Node(19))
tree.insert(Node(7))

tree.display()

print(tree.search(12))

tree.delete(5)
tree.display()

File: java/Main2.java
public class Main2 {
    public static void main(String[] args) {
        BinarySearchTree tree = new BinarySearchTree();

        tree.insert(new Node(12));
        tree.insert(new Node(5));
        tree.insert(new Node(2));
        tree.insert(new Node(3));
        tree.insert(new Node(19));
        tree.insert(new Node(7));
        System.out.println(tree.search(7));
        tree.display();
        tree.remove(5);
        tree.display();
        System.out.println(tree.sum());
    }
}